test/browser-fixtures.test.js

```diff
@@ -139,6 +140,28 @@ describe('PDFMerger', () => {
     expect(diff).toBeFalsy()
   })
 
+  test('merge pdfs from urls', async () => {
```

Comment:
Could you please add some Tests to also test `ArrayBuffer`, `Blob`, `data-url` and some invalifd inputs like strings that are no valid urls and maybe even some nonsence-input like numbers and objects, to test the error-path!? Thx!

Reply:
- Added tests for ArrayBuffer and Buffer.
- Added a mock fetch function since node-fetch and isomorphic-fetch both cause open handle issues with jest, apparently. All of their documentation and SO threads I could find say to just mock fetch instead.
- Added tests for mock Blobs
- Cannot test dataurls or blob urls as those are browser only. jsdom does not currently support simulating them and has no plan to.